src/env.h

```diff
@@ -596,6 +596,11 @@ class Environment {
     DISALLOW_COPY_AND_ASSIGN(TickInfo);
   };
 
+  enum Flags {
```

Comment:
Nit: `Flags` seems to be a bit too generic. It would be less ambiguous if this is something like `enum ThreadType`.

Reply:
@addaleax Sure, we can see how this grows